dataactcore/interfaces/function_bag.py

```diff
@@ -647,3 +647,21 @@ def get_last_validated_date(submission_id):
     # Still need to do a check here in case there aren't any jobs for a submission.
     # This is the case for a single unit test
     return oldest_date.strftime('%m/%d/%Y') if oldest_date else oldest_date
+
+
+def get_action_dates(submission_id):
+    sess = GlobalDB.db().session
+
+    min_action_date = None
+    max_action_date = None
+    date_from_format = '%Y%m%d'
+    date_to_format = '%Y-%m-%d'
+    action_dates = sess.query(func.min(DetachedAwardFinancialAssistance.action_date).label("min_action_date"),
+                              func.max(DetachedAwardFinancialAssistance.action_date).label("max_action_date"))\
+        .filter(DetachedAwardFinancialAssistance.submission_id == submission_id)
```

Comment:
Do we want valid and invalid (has errors) rows to count towards the action dates or only valid rows? Might need to check with someone on this one.